app/services/collators/regular_outgoings_collator.rb

```diff
@@ -11,66 +11,67 @@
 #
 module Collators
   class RegularOutgoingsCollator
-    Attrs = Data.define(:attrs, :child_care_regular)
-    Result = Data.define(:child_care_regular)
+    Attrs = Data.define(:attrs, :child_care_regular, :rent_or_mortgage_regular)
+    Result = Data.define(:child_care_regular, :rent_or_mortgage_regular) do
+      def self.blank
+        new(child_care_regular: 0, rent_or_mortgage_regular: 0)
+      end
+    end
 
     class << self
       def call(disposable_income_summary:, gross_income_summary:, eligible_for_childcare:)
-        new(disposable_income_summary:, gross_income_summary:, eligible_for_childcare:).call
+        attrs = disposable_income_attributes(disposable_income_summary:, eligible_for_childcare:, gross_income_summary:)
+        disposable_income_summary.update!(attrs.attrs)
+        Result.new(child_care_regular: attrs.child_care_regular, rent_or_mortgage_regular: attrs.rent_or_mortgage_regular)
       end
-    end
 
-    def initialize(disposable_income_summary:, gross_income_summary:, eligible_for_childcare:)
-      @disposable_income_summary = disposable_income_summary
-      @gross_income_summary = gross_income_summary
-      @eligible_for_childcare = eligible_for_childcare
-    end
+    private
 
-    def call
-      attrs = disposable_income_attributes
-      @disposable_income_summary.update!(attrs.attrs)
-      Result.new(child_care_regular: attrs.child_care_regular)
-    end
+      def outgoing_categories
+        %i[maintenance_out legal_aid].freeze
```

Comment:
@starswan What do you think of setting up constant for outgoing categories? Eg: CFEConstants::VALID_REGULAR_OUTGOING_CATEGORIES

Reply:
I'm expecting these 2 constants to disappear real soon now - so I'm happy with the duplication as I feel it's clearer which items are still being done like this